nrfx/hal/nrf_gpio.h

```diff
@@ -542,9 +542,11 @@ NRF_STATIC_INLINE NRF_GPIO_Type * nrf_gpio_pin_port_decode(uint32_t * p_pin)
         default:
             NRFX_ASSERT(0);
```

Comment:
Thinking about this again, you could have added an `/* UNREACHABLE */` here instead?

Reply:
But `NRFX_ASSERT` may be empty.